docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,98 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+The `rewrite-table-path` procedure prepares an Iceberg table for moving or copying to another location.
+
+### `rewrite-table-path`
+
+Stages a copy of the Iceberg table's metadata files where every absolute path source prefix is replaced by the specified target prefix.  
+This can be the starting point to fully or incrementally move or copy an Iceberg table to a new location.
+
+!!! info
+    This procedure only stages rewritten metadata files and prepares a list of files to copy. The actual file copy is not part of this procedure.
+
+
+| Argument Name      | Required? | default                                        | Type   | Description                                                             |
+|--------------------|-----------|------------------------------------------------|--------|-------------------------------------------------------------------------|
+| `table`            | ✔️        |                                                | string | Name of the table                                                       |
+| `source_prefix`    | ✔️        |                                                | string | The existing prefix to be replaced                                      |
+| `target_prefix`    | ✔️        |                                                | string | The replacement prefix for `source_prefix`                              |
+| `start_version`    |           | first metadata.json in table's metadata log    | string | The name or path to the chronologically first metadata.json to rewrite  |
+| `end_version`      |           | latest metadata.json                           | string | The name or path to the chronologically last metadata.json to rewrite   |
+| `staging_location` |           | new directory under table's metadata directory | string | The output location for newly modified metadata files                   |
+
+
+#### Modes of operation:
+
+- Full Rewrite:
+
+By default, the procedure operates in full rewrite mode, rewriting all reachable metadata files. This includes metadata.json, manifest lists, manifests, and position delete files.
+
+- Incremental Rewrite:
+
+If `start_version` is provided, the procedure will only rewrite metadata files created between `start_version` and `end_version`. `end_version` defaults to the latest metadata.json of the table. 
```

Comment:
Hm i just noticed you only mention 'end_version' default, and not 'start_version'.

How about:  
```
The 'start_version' and/or 'end_version' arguments may be provided to limit the scope to prepare an incremental copy.  Only metadata files added between 'start_version' and 'end_version' will be rewritten, and only files added in this range are returned in the copy plan.
```